### PR TITLE
Add documentation about EMS Distribution Reports

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/tasks.md [hunk 1]
```diff
@@ -4,27 +4,24 @@ A list of reusable Cumulus tasks. [Add your own.](adding-a-task.md)
 
 ## Tasks
 
-### [@cumulus/discover-granules](https://github.com/cumulus-nasa/cumulus/tree/master/tasks/discover-granules)
-Discover Granules in FTP/HTTP/SFTP endpoints
+### @cumulus/discover-granules
+Discover Granules in FTP/HTTP/HTTPS/SFTP/S3 endpoints
 
-- Schemas: See this module's [schema definitions](https://github.com/cumulus-nasa/cumulus/tree/master/tasks/discover-granules/schemas).
```
Comment (laurenfrederick) [inline docs/tasks.md]: Why did you remove the schemas and links?

File: docs/ems_reporting.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Distribution
+
+Cumulus reports all data distribution requests that pass through the
+distribution API to EMS. In order to track these requests, S3 Server Access
+Logging must be enabled on all protected buckets.
+
+You will need to enable logging for each bucket manually before distribution
+logging will work.
+
+[How Do I Enable Server Access Logging for an S3 Bucket?](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/server-access-logging.html)
```
Comment (laurenfrederick) [inline docs/ems_reporting.md]: We should link to this in the create buckets section.

File: docs/ems_reporting.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# EMS reporting
+
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+TODO - add content here
+
+## Distribution
```
Comment (laurenfrederick) [inline docs/ems_reporting.md]: Should probably add something about the config that goes in the app/config.yml for this

File: docs/ems_reporting.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# EMS reporting
+
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+TODO - add content here
```
Comment (laurenfrederick) [inline docs/ems_reporting.md]: Maybe just get a quick blurb from @jennyhliu on this so we can just wrap this stuff in. i.e. we have 3 reports, this is what they are, this is what you configure. Doesn't have to be in depth.

**Review by laurenfrederick — CHANGES_REQUESTED** [review]

File: docs/tasks.md (hunk 1 above)
Comment (yjpa7145) [reply docs/tasks.md]: Building the docs was failing because the repository URL was not defined for some of the tasks.  I added a conditional to the build script that says, "if there is not a repository URL, don't add things that depend on that repository URL".  Any tasks that are missing a schema or other link are missing it because they aren't defining the homepage or repository URL in their package.json.

File: docs/ems_reporting.md (hunk 3 above)
Comment (yjpa7145) [reply docs/ems_reporting.md]: Once we moved the log location and reports target to the internal bucket, there were no changes to `app/config.yml`.

File: docs/ems_reporting.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+Cumulus creates three ingest related reports to EMS, they are Ingest, Archive and Archive Delete reports.
+
+The Ingest report contains records of all granules, products, or files that have been ingested into CUMULUS.
+
+The Archive report contains recors of all granules, products, or files that have been archived into CUMULUS.  It's similar to Ingest report.
```
Comment (laurenfrederick) [inline docs/ems_reporting.md]: records misspelled. 

should be: 'similar to the ingest report'

File: docs/ems_reporting.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# EMS reporting
+
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+Cumulus creates three ingest related reports to EMS, they are Ingest, Archive and Archive Delete reports.
```
Comment (laurenfrederick) [inline docs/ems_reporting.md]: reword to 'Cumulus creates three ingest related reports for EMS: Ingest, Archive and Archive Delete.'

**Review by laurenfrederick — APPROVED** [review]
